src/components/global-header/global-header.stories.tsx

```diff
@@ -152,6 +182,7 @@ export const GlobalLocalNavBarLayout: ComponentStory<
     </>
   );
 };
+GlobalLocalNavBarLayout.storyName = "Global + Local Nav Bar Layout";
 GlobalLocalNavBarLayout.parameters = {
   docs: { inlineStories: false, iframeHeight: 250 },
 };
```

Comment:
**question(non-blocking)**: do we need to make this an inline story given we are wrapping the component in a containing block via the decorator on line 29?